#16 use $SHELL in combination with GOOS

mrserverless opened the pull request:
to pinpoint PowerShell usage on Windows.

Originally I tried using `os.PathSepartor`, which works on Windows Ubuntu bash but not `MINGW` which defers to the underlying OS which is still windows.

`$SHELL` works better because regardless of the underlying OS, all `bash` shells always have the `$SHELL` environment variable set.

Commit:
#16 use $SHELL to distinguish bash running on Windows.

ejholmes commented on main.go:
```diff
@@ -45,14 +48,15 @@ func main() {
 		format   = flag.String("format", defaultFormat(), "Format can be 'bash' or 'powershell'.")
 	)
-
+	fmt.Printf("format: %s\n", defaultFormat())
```
Is this meant to still be here? Looks like debugging?

ejholmes reviewed (changes requested):
Other than the stray fmt.Printf, this seems like a good way to check for powershell.

Commit:
#16 remove debug statement

mrserverless commented:
Oops sorry, missed that one. Removed! 

Originally I wanted to write some unit tests instead of `fmt.Println` everywhere. But then realised the unit tests would only pass conditionally based on the runner's OS and shell...

Btw for all the Windows users out there, I've added this tool into `scoop.sh`: https://github.com/lukesampson/scoop-extras/pull/525 will raise a PR to update the README soon.

ejholmes commented:
Looks great! Thanks for the change.
